### ISSUE TITLE
Transfer image

### ISSUE THREAD

**Issue by 32nguyen** [fchollet/keras#6991]
I would like to find an example of a training model that convert 1 image to another image. There are some "connections" between those images I want the model to learn. I would like to know if anyone know the example for that.

As I guess the loss should be MSE. I try my model but not have good prediction. 

Thank in advance.
Thanh
